doc/index.rst

```diff
@@ -7,6 +7,19 @@ Introduction
 partup is a program used for initializing flash memory on a target device using
 a predefined layout.
 
+Main Features
+-------------
+
+- Partition block devices
+- Automatic partition size calculation to use whole device
+- Supported disklabels: gpt, msdos
+- Supported filesystems: fat32, ext2, ext3, ext4
```

Comment:
The labels and filesystems could be typed as inline literals as they are directly usable values in the layout config:
```rst
Supported disklabels: ``gpt``, ``msdos``
```